akka-remote-tests/src/multi-jvm/scala/akka/remote/artery/MaxThroughputSpec.scala

```diff
@@ -55,6 +54,12 @@ object MaxThroughputSpec extends MultiNodeConfig {
            # See akka-remote-tests/src/test/resources/aeron.properties
            #advanced.embedded-media-driver = off
            #advanced.aeron-dir = "target/aeron"
+           
+           #advanced.compression {
+           #  enabled = on
+           #  actor-refs.enabled = on
+           #  manifests.enabled = on
+           #}
          }
        }
```

Comment:
how does the advertisement work for this test? we want to run benchmarks when tables are in place